### ISSUE TITLE
Sample project crashes when swiping cards

### ISSUE THREAD

**Issue by samvermette** [Yalantis/Koloda#172]
First the example is throwing a compiler error saying that `countOfVisibleCards` doesn't exist. It seems that this property is now private. So I just took out that line to make the project build.

But then when I swipe the card, or tap x or ❤️ it crashes with the following trace. Not sure if the example is up to date, but any idea?

```
2016-07-02 13:23:54.577 Koloda_Example[54912:4338664] *** Terminating app due to uncaught exception 'CALayerInvalidGeometry', reason: 'CALayer position contains NaN: [nan nan]'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000102b42d85 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x0000000104faedeb objc_exception_throw + 48
	2   CoreFoundation                      0x0000000102b42cbd +[NSException raise:format:] + 205
	3   QuartzCore                          0x0000000105a62e4a _ZN2CA5Layer12set_positionERKNS_4Vec2IdEEb + 152
	4   QuartzCore                          0x0000000105a62fbd -[CALayer setPosition:] + 44
	5   QuartzCore                          0x0000000105a63624 -[CALayer setFrame:] + 650
	6   UIKit                               0x0000000103ad5793 -[UIView(Geometry) setFrame:] + 356
	7   Koloda                              0x00000001028ff158 _TFC6Koloda17DraggableCardViews5frameVSC6CGRect + 424
	8   Koloda                              0x00000001028fef95 _TToFC6Koloda17DraggableCardViews5frameVSC6CGRect + 117
	9   pop                                 0x0000000102973711 _ZL13_staticStates_block_invoke_82 + 273
	10  pop                                 0x0000000102985468 _ZL16updateAnimatableP11objc_objectP26_POPPropertyAnimationStateb + 2152
	11  pop                                 0x0000000102981a74 _ZL18applyAnimationTimeP11objc_objectP18_POPAnimationStated + 260
	12  pop                                 0x0000000102980d1a -[POPAnimator _renderTime:item:] + 906
	13  pop                                 0x0000000102980462 -[POPAnimator _renderTime:items:] + 1362
	14  pop                                 0x000000010298427c -[POPAnimator renderTime:] + 92
	15  pop                                 0x0000000102984210 -[POPAnimator render] + 80
	16  QuartzCore                          0x0000000105a90bf4 _ZN2CA7Display15DisplayLinkItem8dispatchEv + 50
	17  QuartzCore                          0x0000000105a90abe _ZN2CA7Display11DisplayLink14dispatch_itemsEyyy + 418
	18  CoreFoundation                      0x0000000102a9c074 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 20
	19  CoreFoundation                      0x0000000102a9bc21 __CFRunLoopDoTimer + 1089
	20  CoreFoundation                      0x0000000102a5db11 __CFRunLoopRun + 1937
	21  CoreFoundation                      0x0000000102a5d0f8 CFRunLoopRunSpecific + 488
	22  GraphicsServices                    0x0000000106bafad2 GSEventRunModal + 161
	23  UIKit                               0x0000000103a32f09 UIApplicationMain + 171
	24  Koloda_Example                      0x000000010284d672 main + 114
	25  libdyld.dylib                       0x0000000109bf392d start + 1

```

**Comment by Roman-Scherbakov** [comment]
Fixed with #176

**Comment by samvermette** [comment]
Thank you!
